### PR TITLE
Documentation 

### PR DESCRIPTION
**Pull request by schittlur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,187 @@
...
+    - [_REACT_APP_BACKEND_HOST_](#react-app-backend-host)
+    - [_REACT_APP_ES_API_](#react-app-es-api)
+    - [_REACT_APP_ES_MULT_API_](#react-app-es-mult_api)
+    - [_REACT_APP_AUTH_](#react-app-auth)
+    - [_REACT_APP_CONFIGURATION_API_](#react-app-configuration-api)
+    - [_REACT_APP_EXPERIENCES_](#react-app-experiences)
+    - [_REACT_APP_ADD_COURSE_TO_LISTS_](#react-app-add-course-to-lists)
+    - [_REACT_APP_INTEREST_LISTS_ALL_](#react-app-interest-lists-all)
+    - [_REACT_APP_USER_SUBSCRIPTION_LISTS_](#react-app-user-subscription-lists)
+    - [_REACT_APP_USER_INTEREST_LISTS_](#react-app-user-interest-lists)
```
Comment (rglasgow-deloitte) [inline README.md:30]: We need to either remove this section or update it to match the .env file

File: README.md [hunk 2]
```diff
@@ -1,10 +1,187 @@
-# Getting Started with Create React App
+# Open LXP
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+[![Version](https://img.shields.io/badge/version-prototype-yellow)](https://github.com/OpenLXP/openlxp-xms-ui)
+[![yarn](https://img.shields.io/badge/yarn-1.22.1-blue)](https://yarnpkg.com/)
+[![license](https://img.shields.io/badge/license-Apache_2.0-green)](https://github.com/OpenLXP/openlxp-xms-ui/blob/main/LICENSE)
+
+[![react](https://img.shields.io/badge/react-17.0.1-61dafb)](https://reactjs.org/)
+[![redux](https://img.shields.io/badge/redux-4.0.5-764abc)](https://redux-toolkit.js.org/)
```
Comment (rglasgow-deloitte) [inline README.md:8]: Double check to see if we are using redux

File: README.md [hunk 3]
```diff
@@ -1,10 +1,187 @@
...
+http://<YOUR_BACKEND_ENDPOINT>/
+```
+
+#### **Endpoint Name**
+
+Description of the endpoint
+
+```yaml
+http://<YOUR_BACKEND_ENDPOINT>/
+```
```
Comment (rglasgow-deloitte) [inline README.md:139]: Needs to be updated with the relevant info. The vars described should match the .env

File: README.md [hunk 4]
```diff
@@ -1,10 +1,187 @@
...
+
+### Creating a local environment file
+
+#### **Walk through**
+
+Let's create a local `.env.local` file. If you are in a code editor you can right click and create new file.
+
+![vscode create new file](./readme-assets/vscode-create-new-file.png)
```
Comment (rglasgow-deloitte) [inline README.md:147]: Needs the file from our XDS project

File: README.md [hunk 5]
```diff
@@ -1,10 +1,187 @@
...
+[![router](https://img.shields.io/badge/router-5.2.0-red)](https://reactrouter.com/web/guides/quick-start)
+[![tailwind](https://img.shields.io/badge/tailwind-2.2.2-22d3ee)](https://redux-toolkit.js.org/)
+
+## Table of content
+
+- [**Installation**](#installation)
+  - [Clone Project](#clone-project)
+- [**Getting started**](#getting-started)
+  - [Installation](#installation)
+    - [_Clone Project_](#clone-project)
```
Comment (rglasgow-deloitte) [inline README.md:18]: Remove line 18 and keep line 15. 
Links to the same place.

**Review by rglasgow-deloitte — CHANGES_REQUESTED** [review]

**Review by rglasgow-deloitte — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,10 +1,145 @@
...
+```text
+REACT_APP_XIS_CATALOGS_API=<YOUR_BACKEND_ENDPOINT>/api/catalogs/
+REACT_APP_XIS_COMPOSITELEDGER_API=<YOUR_BACKEND_ENDPOINT>/api/metadata/
+```
+
+### Important Notes
+
+To use this piece of code without any issues you will need the XDS component to accompany it.
+
+[OpenLXP XMS](https://github.com/OpenLXP/openlxp-xms)
```
Comment (KarenAJ) [inline README.md:134]: Just checking did you mean to tag XMS here?

**Review by KarenAJ — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (rglasgow-deloitte) [reply README.md:134]: Ill get that, Thanks!

Comment (rglasgow-deloitte) [reply README.md:134]: Done

**Pull request merged**